pkg/manager/member/ticdc_member_manager.go

```diff
@@ -320,6 +320,12 @@ func getNewTiCDCStatefulSet(tc *v1alpha1.TidbCluster) (*apps.StatefulSet, error)
 		}
 	}
 
+	for _, tlsClientSecretName := range tc.Spec.TiCDC.TLSClientSecretNames {
+		ticdcContainer.VolumeMounts = append(ticdcContainer.VolumeMounts, corev1.VolumeMount{
+			Name: tlsClientSecretName, ReadOnly: true, MountPath: fmt.Sprintf("/var/lib/source-tls/%s", tlsClientSecretName),
```

Comment:
How about given a better path part rather than `source` because it's a secret for the downstream?

and it seems you are using `/var/lib/cluster-client-tls/` in PR desc.

Reply:
What about downstream-tls? `/var/lib/cluster-client-tls/` is tc TLS mount path.

Reply:
And just define a const as we're doing here https://github.com/pingcap/tidb-operator/blob/master/pkg/manager/member/ticdc_member_manager.go#L39.

Reply:
@handlerww how about using `sink-tls` because TiCDC use `sink-uri` as the downstream URI.

Reply:
LGTM